app/src/main/java/com/example/habithelper/fragments/ProfileFragment.java

```diff
@@ -28,4 +50,81 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         // Inflate the layout for this fragment
         return inflater.inflate(R.layout.fragment_profile, container, false);
     }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        cText = view.findViewById(R.id.cText);
```

Comment:
I believe we can still improve naming variables. 
Adding the prefix like cText and clMain is common. Code bases usually have their naming conventions and these will vary from project to project. But some of these are to short and do not tell what they are referring, specially since from a diff perspective we don't have intellisense to highlight the type

For example btnChangeZip does imply btn is a button . But cText can be interpreted as currentText, commonText,  checkboxText, the naming is general and as a reviewer I have to actually search for the actual type